src/elements/Select.tsx

```diff
@@ -89,10 +91,11 @@ const hideDefaultSkin = css`
   }
 `
 
-const caretArrow = css`
+const caretArrow = css<SelectProps>`
```

Comment:
Hmm... this works now? Nice. Is that a typescript 3 thing? 🤔

Reply:
I'm not sure! I get a type error without it and followed existing patterns, though we haven't had to pass a type argument to `css` exactly like this before. Perhaps @damassi could chime in?

Reply:
Yup this works with latest styled components!